src/test/java/bio/terra/common/db/DatabaseRetryUtilsTest.java

```diff
@@ -1,77 +1,70 @@
 package bio.terra.common.db;
 
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 import java.time.Duration;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.springframework.dao.CannotSerializeTransactionException;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.dao.DataAccessException;
+import org.springframework.dao.PessimisticLockingFailureException;
 
 /** Test for {@link DatabaseRetryUtils} */
 @Tag("unit")
-public class DatabaseRetryUtilsTest {
-  private static final CannotSerializeTransactionException RETRY_EXCEPTION =
-      new CannotSerializeTransactionException("test");
+@ExtendWith(MockitoExtension.class)
+class DatabaseRetryUtilsTest {
+  private static final PessimisticLockingFailureException RETRY_EXCEPTION =
+      new PessimisticLockingFailureException("test");
+  private static final Duration RETRY_SLEEP = Duration.ofMillis(1);
```

Comment:
was there a reason to change this from `CannotSerializeTransactionException` to `PessimisticLockingFailureException`? Is that what's actually thrown?

Reply:
Yes, I changed this because `CannotSerializeTransactionException` is now deprecated. The exception `PessimisticLockingFailureException` is a more general exception that includes `CannotSerializeTransactionException` and is the recommended replacement.

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/dao/CannotSerializeTransactionException.html
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/dao/PessimisticLockingFailureException.html